AsyncDisplayKit/ASVideoNode.mm

```diff
@@ -437,6 +440,24 @@ - (void)setPlayerState:(ASVideoNodePlayerState)playerState
   _playerState = playerState;
 }
 
+- (void)setAssetURL:(NSURL *)assetURL
+{
+  if (!ASObjectIsEqual(assetURL, self.assetURL)) {
+     self.asset = [AVURLAsset assetWithURL:assetURL];
```

Comment:
What if the player item has been created already?  Does the clearFetchedData / fetch data pass cause this to be updated correctly?